app/services/exporter/service.rb

```diff
@@ -26,7 +26,7 @@ def export(send_email: false, config: {})
 
       zipped_exports = zip_exports(exports)
 
-      send_exports_by_email(zipped_exports) if send_email
+      send_exports_by_email(zipped_exports, send_to_admin: send_to_admin)
```

Comment:
Mentioning again that this removes the option to _not_ send an email. I don't think we ever had a case where we did that, except within console.